lib/exposes.js

```diff
@@ -177,6 +177,32 @@ class Light extends Base {
         return this;
     }
 
+    withLevelConfig() {
```

Comment:
Maybe we should only add this for bulbs we know it works with. I can already imagine the amount of issues being created because these won't work for China bulb X

Reply:
What about not exposing it for bulbs at all, but exposing it for devices like the Ubisys D1?
I imagine you want to use this on a D1, but probably not on bulbs? (But it's nice to have the converters hooked up in case you do)

Reply:
I agree, converters are indeed fine and nothing against in having these by default. Exposes only for D1 sounds good